src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] arg) {
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = new FigureSupplier().getRandomFigure();
+        }
+
+        for (int i = 3; i < figures.length; i++) {
```

Comment:
Consider using `i = figures.length / 2` instead of `i = 3` to make the code more adaptable to changes in the array size.